src/participant/participant.ts

```diff
@@ -648,19 +543,16 @@ export default class ParticipantController {
           }
         )) || [];
 
-      const schema = await getSimplifiedSchema(sampleDocuments);
-      this._schema = new SchemaFormatter().format(schema);
-
-      const useSampleDocsInCopilot = !!vscode.workspace
-        .getConfiguration('mdb')
-        .get('useSampleDocsInCopilot');
+      const unformattedSchema = await getSimplifiedSchema(sampleDocuments);
+      const schema = new SchemaFormatter().format(unformattedSchema);
 
-      if (useSampleDocsInCopilot) {
-        this._sampleDocuments = getSimplifiedSampleDocuments(sampleDocuments);
-      }
+      return {
+        sampleDocuments: getSimplifiedSampleDocuments(sampleDocuments),
```

Comment:
Do we want to parse sample documents even if we know that setting is disabled? Isn't it more performant to only do this if we are going to use those?